tests/pkg/spaces/test.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+rm -rf .lake/build
+lake exe "«path with spaces»"
```

Comment:
```suggestion
lake exe "path with spaces"
```

I think the French quotes here are causing the problem. I skip the part of Lake's `init` test that uses French quotes because I have found they are not properly passed to Lake from the CLI.

https://github.com/leanprover/lean4/blob/073b2cfc83dbb385548b5e5851e69d0713a578dc/src/lake/tests/init/test.sh#L124-L128

Reply:
I believe this is an argument encoded issue, as this example program I think demonstrates:

```
$ echo 'def main (as : List String) : IO Unit := IO.println (repr as)' | lean --stdin --run "«path with spaces»"
[]

Reply:
I think that's a different `--run` bug, at least on Linux. Could you try
```
echo 'def main (as : List String) : IO Unit := IO.println (repr as)' | lean --stdin --run '«path with spaces»' '«path with spaces»'
```
?

Reply:
#4601